<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addreaderDialog</class>
 <widget class="QDialog" name="addreaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>397</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>基本信息</string>
    </attribute>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>248</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
alternate-background-color: rgb(13, 109, 193);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_dataset">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/calenda.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>8</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>卡号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>性别：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>出生年月：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>专业：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>36</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>身份：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_message">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_rt">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>读者类型：</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>lineEdit_1</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>lineEdit_3</zorder>
    <zorder>lineEdit_4</zorder>
    <zorder>lineEdit_6</zorder>
    <zorder>lineEdit_7</zorder>
    <zorder>lineEdit_5</zorder>
    <zorder>pushButton_dataset</zorder>
    <zorder>label_message</zorder>
    <zorder>comboBox_rt</zorder>
    <zorder>label_8</zorder>
    <zorder>calendarWidget</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>图像信息</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>图像预览区</string>
     </property>
     <widget class="QLabel" name="label_headpict">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/logininback.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>获取头像</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_capture">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>181</y>
       <width>401</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_photopath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_upload">
        <property name="text">
         <string>本地上传</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_takepicture">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>拍照</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Capture">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>获取头像</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
